sockeye/arguments.py

```diff
@@ -52,6 +52,22 @@ def parse(schedule_str):
         except ValueError:
             raise argparse.ArgumentTypeError("Learning rate schedule string should have form rate1:num_updates1[,rate2:num_updates2,...]")
         return schedule
+
+    return parse
+
+
+def two_ints(greater_or_equal: Optional[int] = None) -> Callable:
```

Comment:
If we made this more general (int_list or similar), we could fold in things like number of filters for the convolutional embedding encoder:

    --conv-embed-num-filters 200:200:250:250:300:300:300:300